docs/DeveloperGuide.md

```diff
@@ -659,86 +669,170 @@ Given below are instructions to test the app manually.
 
 </div>
 
 ## 7.1 Launch and shutdown
 
 1. Initial launch
 
-    1. Download the `ConnectUS.jar` file and copy into an empty folder
+   1. Download the `ConnectUS.jar` file and copy into an empty folder
 
-    1. Double-click the `ConnectUS.jar` file Expected: Shows the <u>GUI</u> with a set of sample contacts. The window size may not be optimum.
+   1. Double-click the `ConnectUS.jar` file Expected: Shows the <u>GUI</u> with a set of sample contacts. The window size may not be optimum.
 
 2. Saving window preferences
 
-    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-    1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
-
-3. _{ more test cases …​ }_
+   1. Re-launch the app by double-clicking the jar file.<br>
+      Expected: The most recent window size and location is retained.
 
 [↑ Back to top of section](#7-instructions-for-manual-testing)
 
 [↑ Back to table of contents](#table-of-contents)
 
 <div style="page-break-after: always"></div>
 
-## 7.2 Adding a person:
-1. Adding a `Person` with just name and email
-    1. Prerequisites: None
-    2. Test case: `add n/JohnDoe e/email@example.com`<br>
-       Expected: a new `Person` named JohnDoe with given email is  created. Details of the new `Person` shown in the status message. `Person` is visible in contact list.
-    3. Test case: `add n/ e/email@example.com`<br>
-       Expected: No `Person` is created. Error details shown in status message.
-    4. _{ more test cases …​ }_
+## 7.2 Adding a contact:
+
+1. Adding a `Person` with just name, email and phone
+   1. Test case: `add n/John Doe e/email@example.com p/88291322`<br>
+      Expected: a new `Person` named JohnDoe with given email and phone
+      number is created. Details of the new `Person` shown in the status message. `Person` is visible in contact list.
+   2. Test case: `add n/ e/email@example.com`<br>
+      Expected: No `Person` is created. Error details shown in status message.
+2. Adding a `Person` with additional fields such as address, birthday, social medias etc. (For a detailed list see [this](https://ay2223s2-cs2103t-w15-1.github.io/tp/UserGuide.html#5-information-fields--prefixes))
+   1. Test case: `add n/Peter Davis e/peter@example.com p/92849132 b/11/09/1989 a/Road No. 12, Kent Ridge, Singapore ig/peterdavid`. Expected a new `Person` named Peter Davis with given email, phone number, birthday, address and Instagram handle is created. Details of the new `Person` shown in the status message. `Person` is visible in contact list.
+  
+
 
 [↑ Back to top of section](#7-instructions-for-manual-testing)
 
 [↑ Back to table of contents](#table-of-contents)
 
 <div style="page-break-after: always"></div>
 
+## 7.3 Editing a contact:
```

Comment:
Ditto, person vs contact. Also probably good to keep the spacing between the heading and the content consistent.

e.g. glossary has spacing between heading of letter and content. this section should have spacing as well maybe?